packages/twenty-front/src/modules/object-record/utils/getAvailableFieldsForAggregationFromObjectFields.ts

```diff
@@ -0,0 +1,67 @@
+import { FieldMetadataItem } from '@/object-metadata/types/FieldMetadataItem';
+import { AGGREGATE_OPERATIONS } from '@/object-record/record-table/constants/AggregateOperations';
+import { FieldMetadataType } from '~/generated-metadata/graphql';
+
+type AGGREGATE_OPERATIONS_OMITTING_COUNT = Exclude<
```

Comment:
The naming convention is reserved for const values not types, we should keep PascalCase here.